### ISSUE TITLE
Inconsolata powerline characters don't line up correctly

### ISSUE THREAD

**Issue by acarl005** [ryanoasis/nerd-fonts#992]
#### 🗹 Requirements

- [x] I have searched the [issues](https://github.com/ryanoasis/nerd-fonts/issues) for my issue and found nothing related and/or helpful
- [x] I have searched the [FAQ](https://github.com/ryanoasis/nerd-fonts/wiki/FAQ-and-Troubleshooting) for help
- [x] I have searched the [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki) for help

#### 🎯 Subject of the issue

Experienced behavior:

Powerline characters in Inconsolata Nerd Font do not line up properly. This is happening in multiple terminals, e.g.

Alacritty:

<img width="676" alt="Screenshot 2022-11-04 at 9 29 44 PM" src="https://user-images.githubusercontent.com/8334252/200360600-57e84011-5fa4-456c-934b-23143fda3070.png">

Terminal.app:

<img width="696" alt="Screenshot 2022-11-04 at 9 29 45 PM" src="https://user-images.githubusercontent.com/8334252/200360647-eb475e4f-8012-45d0-9d31-2406010b6bdf.png">

Expected behavior:

The powerline characters should line up as they do with Bitstream Vera Mono Nerd Font:

<img width="720" alt="Screenshot 2022-11-04 at 9 30 44 PM" src="https://user-images.githubusercontent.com/8334252/200360804-47914aca-62a8-4e97-9910-d39b896e3730.png">

Example symbols:

```
 andy  ﱮ ~/s/ls-go    master []   v1.19.2 $
```

#### 🔧 Your Setup

- _Which font are you using (e.g. `Anonymice Powerline Nerd Font Complete.ttf`)?_
   - Inconsolata Nerd Font Mono
   - `brew install --cask font-inconsolata-nerd-font`
- Alacritty or Terminal.app (the included terminal with MacOS)
- MacOS Venture 13.0

**Comment by Finii** [comment]
The cask is outdated...

![image](https://user-images.githubusercontent.com/16012374/200376068-528a6933-d532-4b89-8983-44cc0bd7b3b0.png)

You can try to (manually) install the fonts from the (`v2.2.2`) release page:
https://github.com/ryanoasis/nerd-fonts/releases/download/v2.2.2/Inconsolata.zip

Hmm, but even with 2.2.2 they DO look somehow weird, not as bad, but still:

![image](https://user-images.githubusercontent.com/16012374/200377641-d0d0ea83-9f51-4732-9d9d-371a5bde1dc4.png)

----

Link to #989, we need to update a lot of the casks. Who maintains them :grimacing:

**Comment by Finii** [comment]
I, my bad... I used `Inconsolata` which gets the powerline symbols from some other font. When I select the correct font it looks good:

![inconsolata](https://user-images.githubusercontent.com/16012374/200395555-5cf62c89-d24d-497e-9a63-9a39348ad388.gif)

Also checked `kitty`, and there it looks ok.
It = `v2.2.2`.

**Comment by NikitaNaumenko** [comment]
Same issue, I download font from release page https://github.com/ryanoasis/nerd-fonts/releases/download/v2.2.2/iA-Writer.zip and install it manually,
OS : MacOS
Term: Alacritty
 Font: iM Writing Mono S Regular Nerd Font Complete
<img width="803" alt="image" src="https://user-images.githubusercontent.com/18601102/201540518-acf5e198-ea3d-471b-bb09-e88a318a1fb8.png">

But it looks better with iM Writing Mono S Regular Nerd Font Complete Mono
<img width="800" alt="image" src="https://user-images.githubusercontent.com/18601102/201540967-8c1512d6-1d86-4967-a957-1881653cc13d.png">

**Comment by Finii** [comment]
Hmm, can not reproduce...

```
$ curl -LO https://github.com/ryanoasis/nerd-fonts/releases/download/v2.2.2/iA-Writer.zip
$ unzip -d iMwriting iA-Writer.zip
$ cp iMwriting/iM\ Writing\ Mono\ S\ Regular\ Nerd\ Font\ Complete.ttf ~/.local/share/fonts
$ fc-list | grep -i writ                                                                   
/home/fini/.local/share/fonts/iM Writing Mono S Regular Nerd Font Complete.ttf: iMWritingMonoS Nerd Font:style=Regular
```

Edit `./alacritty.yml`:
```
# Font configuration
font:
  normal:
    family: iMWritingMonoS Nerd Font
    style: Regular
```

Run
```
$ ~/git/alacritty/target/release/alacritty --config-file ./alacritty.yml
```

![image](https://user-images.githubusercontent.com/16012374/201866735-b2392c03-2016-4564-9bd3-e971c3769ec0.png)

(The font in `tilix` is Delugia).

My alacritty is not really up to date, it's `alacritty 0.11.0-dev (589c1e9)` I built some months(?) ago.

Sure you entered the correct family name in your config?

Same goes to @acarl005: Please show your alacritty config and relevant `fc-list` output.

**Comment by NikitaNaumenko** [comment]
First of all, thanks for your reply and your help!

I repeat all your steps:
fc-list output

> 
> ➜ fc-list | grep -i writ
> Fontconfig warning: ignoring UTF-8: not a valid region tag
> /System/Library/Fonts/Supplemental/AmericanTypewriter.ttc: American Typewriter:style=Regular
> /Users/nikitanaumenko/Library/Fonts/iM Writing Mono S Regular Nerd Font Complete Mono.ttf: iMWritingMonoS Nerd Font Mono:style=Regular
> /System/Library/Fonts/Supplemental/AmericanTypewriter.ttc: American Typewriter:style=Semibold
> /System/Library/Fonts/Supplemental/AmericanTypewriter.ttc: American Typewriter:style=Condensed Light
> /System/Library/Fonts/Supplemental/AmericanTypewriter.ttc: American Typewriter:style=Condensed Bold
> /System/Library/Fonts/Supplemental/AmericanTypewriter.ttc: American Typewriter:style=Condensed
> /System/Library/Fonts/Supplemental/AmericanTypewriter.ttc: American Typewriter:style=Bold
> /Users/nikitanaumenko/.local/share/fonts/iM Writing Mono S Regular Nerd Font Complete.ttf: iMWritingMonoS Nerd Font:style=Regular
> /System/Library/Fonts/Supplemental/AmericanTypewriter.ttc: American Typewriter:style=Light
> 

My alacritty config is:
<img width="1272" alt="image" src="https://user-images.githubusercontent.com/18601102/201881853-73c78bcf-f5ce-45a5-9f3e-8efeeb861c16.png">

Same screenshot with Mono:
<img width="1273" alt="image" src="https://user-images.githubusercontent.com/18601102/201882117-bd620383-cfc5-4907-bf47-42923b8e1164.png">

**Comment by NikitaNaumenko** [comment]
pfff I dont know what happened, but now it works better, I do nothing, just opened my laptop, and I cannot reproduce it any more

<img width="1023" alt="image" src="https://user-images.githubusercontent.com/18601102/202535258-34e11a89-9ce4-44eb-b253-c44c78e7a31f.png">

**Comment by acarl005** [comment]
Looks perfect for me on all terminal ems after installing v2.2.2 from release page! Thanks @Finii! It looks like the long-term resolution here is for the Homebrew packages to be updated more easily. I understand maintaining a bunch of fresh packages on various different registries is a lot of work..

**Comment by Finii** [comment]
@NikitaNaumenko

> I dont know what happened, but now it works better

It's called _self healing_ ;-)

Thanks for notifying back!

@acarl005 We create the needed files already, it would be not too much work to automate the PRs to update the casks. But every item takes power and time. It definitively increased in prio now.
I tried to list all packag-eteers here: https://github.com/ryanoasis/nerd-fonts/discussions/881#discussioncomment-3488054 and Homebrew is one of the most easy ones to update.
Also thanks for your information.

**Comment by hotsno** [comment]
I'm having this issue on Alacritty and iTerm. I tried v2.3.1 of Meslo and iA-Writer.

<img width="852" alt="image" src="https://user-images.githubusercontent.com/71658949/213637834-1dd855b5-9b01-4f4e-8a1c-0ab80530ba8b.png">

<img width="912" alt="image" src="https://user-images.githubusercontent.com/71658949/213637876-76433f45-d1dc-4bdf-97d8-591d3c264f39.png">

**Comment by Finii** [comment]
Sorry to hear you have problems. 
Thanks for reporting. 
I guess this is #1056, but will check. 



________________________________

**Comment by hotsno** [comment]
I restarted my computer after reading that someone above just had their issue resolved after closing their laptop... and it magically became better now. Maybe a macOS thing.

<img width="159" alt="image" src="https://user-images.githubusercontent.com/71658949/213642098-9b5fa6a0-eb7d-4979-a56b-94ec81e17be1.png">

The "blur" part of my prompt (the leftmost part of my prompt in the screenshots above) is still having issues in iTerm, though. For the time being I just disabled it.
